lib/primo/pnxs.rb

```diff
@@ -79,8 +68,11 @@ def url
       end
 
       def params
-        @params.merge(auth)
-          .merge q: @params[:q].to_s
+        query = @params[:q]
+        p = @params.merge(auth).merge(q: query.to_s)
+        p.merge!(qInclude: query.include_facets) unless query.include_facets.nil?
```

Comment:
I wonder if it makes sense for `query.include_facets` and `query.exclude_facets` to return a has that is either empty `{}` or  has the required bit `{qInclude: foobar}`; that way merges work in all cases and there is no logic needed for the `nil` case.